jetty-maven-plugin/src/it/javax-annotation-api/pom.xml

```diff
@@ -45,7 +45,12 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>log4j-slf4j12</artifactId>
+      <artifactId>slf4j-log4j12</artifactId>
```

Comment:
The prior entry `log4j-slf4j12` doesn't exist. (it was a mistake of mine in a prior commit on this branch)